ooni/nettests/manipulation/http_invalid_request_line.py

```diff
@@ -45,8 +45,11 @@ def setUp(self):
     def check_for_manipulation(self, response, payload):
         log.debug("Checking if %s == %s" % (response, payload))
         if response != payload:
+            log.msg("Detected manipulation!")
+            log.msg(response)
             self.report['tampering'] = True
         else:
+            log.msg("No manipulation detected.")
             self.report['tampering'] = self.report['tampering'] | False
```

Comment:
unrelated with the diff, but: what's the point of `self.report['tampering'] | False`?